tests/robot-tests/scripts/create_snapshots.py

```diff
@@ -4,7 +4,6 @@
 import requests
 from bs4 import BeautifulSoup
 
-
```

Comment:
Not sure we follow PEP8 everywhere, but it suggests having two blank lines before and after top-level functions: https://www.python.org/dev/peps/pep-0008/#blank-lines